src/index.ts

```diff
@@ -15,6 +15,7 @@ export type Config = {
   options?: ConfigOptions;
 };
 
+// why just pass the argument directly back?
```

Comment:
in the config file, the user could define the needed export object by declaring it and then enforcing the type by importing the ```Config``` type, but I prefer exporting the defineConfig function as the default export to mark it clearly as the preferred interface